app/scripts/controllers/monitoring.js

```diff
@@ -128,6 +147,18 @@ angular.module('openshiftConsole')
       $scope.logCanRun.builds[build.metadata.name] = !(_.includes(['New', 'Pending', 'Error'], build.status.phase));
     };
 
+    var filterStatefulSets = function() {
+      ageFilteredStatefulSets = _.filter($scope.statefulSets, function(set) {
+        if (!$scope.filters.hideOlderResources) {
+          return true;
+        }
+        // TODO: statefulSets don't have status.phase. As far as I understand, they don't age the
+        // way other things age, so perhaps this is irrelevant?
```

Comment:
yep i think you are right, I don't think we would hide any statefulsets ever, so you don't need to change anything when the filter is checked/unchecked